examples/is-like/is_like.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/io;
+
+type Person record {
+    string name = "";
+    int age = 0;
+};
+
+type Employee record {
+    string name = "";
+    int age = 0;
+    int empNo = 0;
+};
+
+// Concept is-like contains `convert()` and `stamp()` functions. is-like is about converting runtime value's inherent storage type.
```

Comment:
I think we need to break the lines at col 80. Can you check?

Reply:
File was removed